spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  let(:user) { create(:user) }
+  let(:other) { create(:user) }
+
+  before(:each) do
+    allow(controller).to receive(:current_user) { user }
+  end
+
+  allow_user_to :manage, User
+
+  describe 'GET #show' do
+    it 'assigns the requested user as @user' do
+      get(:show, id: other.to_param)
+      assigns(:user).should eq(other)
+    end
+  end
+
+  describe 'GET #edit' do
+    it 'should render edit page' do
+      get(:edit)
+      response.code.should eq("200")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.